src/components/MintSong/MintSong.tsx

```diff
@@ -144,17 +152,17 @@ const MintSong: React.FC = () =>{
             walletContext.walletAddress,
             0
           );
-          // end minting
-        //   setIsMinting(false);
+        //   end minting
+          setIsMinting(false);
```

Comment:
This is the part the shows the loading indicator, but I will change it to show different steps of the minting process like you once stated. I have some design changes to make first